src/tests/Microsoft.PowerFx.Core.Tests/PublicSurfaceTests.cs

```diff
@@ -178,7 +178,8 @@ public void PublicSurface_Tests()
                 "Microsoft.PowerFx.Core.RenameDriver",
                 "Microsoft.PowerFx.Core.Utils.DName",
                 "Microsoft.PowerFx.Core.Utils.DPath",
-                "Microsoft.PowerFx.Core.Utils.ICheckable",                
+                "Microsoft.PowerFx.Core.Utils.ICheckable",
+                "Microsoft.PowerFx.IRuntimeConnectorContext",

```

Comment:
>IRuntimeConnectorContext [](http://example.com/codeflow?start=35&length=24)

Let's find a way to move this up out of core.

Reply:
Let's talk